windows-it-install.ps1

```diff
@@ -0,0 +1,32 @@
+# Following script works with the Windows 2016 image provided by GCE. Script may not work for other images/versions of Powershell.
```

Comment:
Suggest adding line-breaks here to enhance comment readability.

Reply:
done